config/application.rb

```diff
@@ -307,6 +307,10 @@ class Application < Rails::Application
         set_dependent_action :destroy
         template_compatibility_properties :cores, :virtual, :sockets, :ram, :uptime_seconds
       end
+      Facets.register(HostFacets::InfrastructureFacet, :infrastructure_facet) do
+        api_view({ :list => 'api/v2/hosts/infrastructure_facet' })
```

Comment:
Is it good practice to use a URL helper method for the path here?

Reply:
Afaik this is not a part of an url, but a path under `app/views`. Maybe there's an url helper which would match it, but I'd rather stay away from it.